apps/batclock/README.md

```diff
@@ -1,13 +1,14 @@
 <div align="center">
 <h1> BatClock </h1>
 <p align="center">
-  <img src="./assets/screenshot.png" width="240px" height="240px">
+  <img src="https://github.com/espruino/BangleApps/raw/master/apps/batclock/assets/screenshot.png" height="240px">
```

Comment:
Please could you just use `src="screenshot.png"` here? That should be all you need...

Reply:
Thanks, Done. Update at [here](https://github.com/espruino/BangleApps/pull/725) as well